### PR TITLE
Fixed #30594 -- Added 'private' Cache-Control directive to never_cache() decorator.

### PR DESCRIPTION
**Pull request by nsasaki128**
Ticket: https://code.djangoproject.com/ticket/30594#ticket

### CODE REVIEW

File: docs/topics/http/decorators.txt [hunk 1]
```diff
@@ -119,5 +119,5 @@ client-side caching.
 .. function:: never_cache(view_func)
 
     This decorator adds a ``Cache-Control: max-age=0, no-cache, no-store,
-    must-revalidate`` header to a response to indicate that a page should never
+    must-revalidate, private`` header to a response to indicate that a page should never
```
Comment (felixxm) [inline docs/topics/http/decorators.txt]: Please wrap at 79 chars and add `versionchanged`, e.g.

```
    .. versionchanged:: 3.0

        ``private`` directive was added.
```

Comment (nsasaki128) [reply docs/topics/http/decorators.txt]: Thanks for the comment!
I'll fix it.

**Comment by felixxm** [comment]
@nsasaki128 Thanks! I fixed `add_never_cache_headers()` documentation.

**Comment by nsasaki128** [comment]
@felixxm Thanks a lot for fixing. Django helps our products development and I am honored to be involved.

**Comment by felixxm** [comment]
@nsasaki128 Welcome aboard :motor_boat:

**Pull request merged**